code/testapp/build.gradle

```diff
@@ -52,7 +52,6 @@ dependencies {
     implementation 'com.adobe.marketing.mobile:identity:2.+'
     implementation 'com.adobe.marketing.mobile:assurance:2.+'
 
-
```

Comment:
This file was recently updated by another PR. You can remove this change and merge in the latest from `dev` branch.